packages/multi-test/src/lib.rs

```diff
@@ -6,6 +6,8 @@
 //!
 //! To understand the design of this module, please refer to `../DESIGN.md`
 
+#![allow(clippy::type_complexity)]
```

Comment:
I understand this is needed on all three modules below, right?

Reply:
No, just `contracts`. We can make it more granular if we want to. The lint seems like it's sometimes broken though:

https://github.com/rust-lang/rust-clippy/issues/1013#issuecomment-587054810

Reply:
Let's do that if possible, yes. That will give us the opportunity to at least flag upcoming and take a look at future type complexities.

Reply:
That's fair, done.